### ISSUE TITLE
example data

### ISSUE THREAD

**Issue by holgerbrandl** [julipetal/petalNet#2]
The repository should contain an example data, so that new users can get started more easily. 

The paper says that the input must be tab-separated but does not indicate how gene-names and samples should be encoded (row-names, column-names, header). Also it's not clear if the goi-input file needs some header or is a plain list.

**Comment by ghost** [comment]
I also agree with this commet and I'm looking forward to some hints as I am also keen to try petal with my RNA-Seq data.

**Comment by julipetal** [comment]
I apologize, I am still putting a tutorial together. 
I added example input files.

**Comment by thkitapci** [comment]
Hi,
I would like to try running petal with a dataset I have but I am unable to run the sample inputs in the repository. There are 2 sample input files in the repository but I don't understand how to run Petal with these inputs. I don't see a sample goi-input file. Can you please include the required input files to run the dataToVNs() function ? It would be nice if you can also include a sample annotation file to see how the plots look for the sample data with annotations. 

Thanks
